package.json

```diff
@@ -62,6 +62,7 @@
     },
     "dependencies": {
       "react": "npm:react@^16.2.0",
+      "react-addons-css-transition-group": "npm:react-addons-css-transition-group@^15.6.2",
```

Comment:
The group that made this plugin recommend that you use "react-transition-group" instead, which is a newer (although not backwards-compatible) version.  https://github.com/reactjs/react-transition-group